redis/src/main/scala/zio/redis/ClusterExecutor.scala

```diff
@@ -150,9 +150,11 @@ object ClusterExecutor {
     } yield ExecutorScope(executor, closableScope)
 
   private def redis(address: RedisUri) = {
-    val executorLayer = ZLayer.succeed(RedisConfig(address.host, address.port)) >>> RedisExecutor.layer
-    val codecLayer    = ZLayer.succeed[BinaryCodec](StringUtf8Codec)
-    val redisLayer    = executorLayer ++ codecLayer >>> Redis.layer
+    val redisConfigLayer = ZLayer.succeed(RedisConfig(address.host, address.port))
+    val codecLayer       = ZLayer.succeed[BinaryCodec](StringUtf8Codec)
+    val executorLayer    = redisConfigLayer >>> RedisExecutor.layer
+    val pubSubLayer      = redisConfigLayer ++ codecLayer >>> RedisPubSub.layer
+    val redisLayer       = executorLayer ++ pubSubLayer ++ codecLayer >>> Redis.layer
```

Comment:
🤔 
Strange that to execute redis simple command we need pub sub layer.
Should it be part of Redis.Live ?

Reply:
@anatolysergeev I separated subscription part of PubSub into `RedisSubscription`
it includes only commands (`subscribe`, `psubscribe`, `unsubscribe`, `punsubscribbe`) that changes protocol resp to pushed type.